composer.json

```diff
@@ -16,12 +16,13 @@
         }
     },
     "require": {
-        "illuminate/translation": "^8.0||^6.0"
+        "php": "^7.2|^8.0",
```

Comment:
Try `7.2|8.0`

Reply:
Hmm, different errors now. Do you think I should specify the PHP version in the CI ?

Reply:
You still left the `^` for 8.0

Reply:
Still failing.

Reply:
@abrardev99 This is still unresolved

Reply:
After this change `7.2|8.0`, Tests failed because CI installed PHP `8.0.14`. I could fix it by requiring the exact version `8.0.14` in composer.json or `^8.0`. I went with the second approach, which solves the issue.

Reply:
`8.0` should work with `8.0.x`. The previous failure was due to a different issue I think.

We can't merge this anyway because even if the CI build happens to pass here, `7.2|^8.0` is weird and inconsistent. So it needs to be fixed.

So try making the change again.

Reply:
Alternatively, do we *even need to add this*? Seems like we didn't specify a PHP version before.

Reply:
> Alternatively, do we even need to add this? Seems like we didn't specify a PHP version before.

Let me try this again on my fork.

Reply:
Looks like we don't need to specify the PHP version in `composer.json`. But we do need to install only PHP version 8 in CI; otherwise, tests on Laravel 6 failed.